### DISCUSSION TITLE
Validate user profiles before they are allowed in the live service part of the app using people that are already on the app.

### DISCUSSION THREAD

**Discussion by WilliamDormer** [PullDating/Org#35] [Ideas & Features]
Make the first 5 swipes of a person when they get on for the day validating new user profiles to tell us if they are scams, spammers, overly horny, abusive etc. This way, we can really cut down on the bad actors on the live part of the app. Then when someone changes their profile, they need to be re-validated. If then they pay, then they don't need to perform the validation action every day. We would have to think about how to prevent people from just saying "yep all these profiles are good" but as a collective effort this could work.

**Comment by WilliamDormer** [comment]
Another option where you use a short video call between users to verify they are who they say they are, which would further minimize the risk.

**Comment by ajbott** [comment]
I think this idea has a lot of potential to help keep the app free and provide a free workforce to uphold community standards.
